src/Components/NavBar/NotificationsBadge.tsx

```diff
@@ -38,61 +40,75 @@ export const NotificationsBadge: React.FC = () => {
           0
         )
 
-        let notifications = totalUnread
+        let count = totalUnread
 
         // Update the notification bad with the count, and store it in a cookie
         // so that subsequent page views don't need a fetch in order to render
         // the badge.
-        if (notifications > 0) {
+        if (count > 0) {
           const cachedNotificationCount = Number(
             cookie.get("notification-count")
           )
-          if (notifications !== cachedNotificationCount) {
-            if (notifications >= 100) {
-              notifications = "99+"
+          if (count !== cachedNotificationCount) {
+            if (count >= 100) {
+              count = "99+"
             }
 
             // In force, when a request is made to `/notifications` endpoint,
             // sd.NOTIFICATIONS_COUNT is populated by this cookie.
-            cookie.set("notification-count", notifications)
+            cookie.set("notification-count", count)
           }
         }
 
         // User has no notifications; clear the cookie
-        if (notifications === 0) {
+        if (count === 0) {
           cookie.expire("notification-count")
           return null
         }
 
-        const trackOnHover = once(() => {
-          trackEvent({
-            subject: AnalyticsSchema.Subject.NotificationBell,
-            new_notification_count: totalUnread,
-          })
-        })
-
         return (
-          <Box onMouseOver={trackOnHover}>
-            <CircularCount notifications={notifications} />
+          <Box>
+            <CircularCount count={count} rawCount={totalUnread} hover={hover} />
```

Comment:
We pass along the raw count in addition to the formatted `count`. Analytics wants the full integer, not the formatted "99+" label string.